Plutarch/Bool.hs

```diff
@@ -89,6 +90,18 @@ class PEq t => POrd t where
 infix 4 #<=
 infix 4 #<
 
+-- | Partial ordering relation.
+class PPartialOrd t where
```

Comment:
Should this become a superclass of `POrd`? It'd require adding all the instances everywhere else